my-sms/Gemfile

```diff
@@ -0,0 +1,43 @@
+source 'https://rubygems.org'
+
+gem 'rails', '~> 3.2.0'
+
+# Bundle edge Rails instead:
+# gem 'rails', :git => 'git://github.com/rails/rails.git'
+
+gem 'mysql2', '0.3.20'
+
+# tooling
+gem 'rubocop'
+gem 'reek'
+gem 'byebug'
+gem 'rspec'
```

Comment:
Good to see rubocop and reek here 👍
I recommend adding these to the development and test groups, as they aren't needed for other environments:

```
group :development, :test do
  gem 'rubocop'
  gem 'reek'
  gem 'byebug'
  gem 'rspec'
end
```

You might also want to consider adding the `rspec-rails` gem

Reply:
I fix this as part of the next task which has the requirement to set up rspec. I can commit it on top of here and rebase if you like.

Reply:
Nah, no need to commit/rebase in here. Just something to consider for the new PR 🙂